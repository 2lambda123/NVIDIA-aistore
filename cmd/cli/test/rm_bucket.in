ais create bucket $BUCKET_1
ais rm bucket $BUCKET_1

ais rm bucket aws://$BUCKET_2 // FAIL "Removing cloud buckets (aws://$BUCKET_2) is not supported"
ais rm bucket cloud://$BUCKET_2 // FAIL "Removing cloud buckets (cloud://$BUCKET_2) is not supported"
ais rm bucket gcp://$BUCKET_2 // FAIL "Removing cloud buckets (gcp://$BUCKET_2) is not supported"
