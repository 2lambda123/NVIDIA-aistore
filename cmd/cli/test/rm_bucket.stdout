"$BUCKET_1" bucket created
"$BUCKET_1" bucket destroyed
